.eslintrc

```diff
@@ -29,31 +32,18 @@
     "jsdoc/no-undefined-types": "off",
     "jsdoc/require-returns-type": "off",
     "prettier/prettier": "error",
-    "@typescript-eslint/naming-convention": [
+    "no-use-before-define": "off",
+    "no-useless-escape": "off",
+    "spaced-comment": "error",
+    "curly": [
       "error",
-      {
-        "format": ["PascalCase"],
-        "selector": "typeLike",
-        "custom": {
-          "regex": "^I[A-Z]",
-          "match": false
-        }
-      }
+      "multi-line"
     ],
-    "@typescript-eslint/member-ordering": "error",
-    "no-use-before-define": "off",
-    "@typescript-eslint/no-use-before-define": [
+    "eol-last": [
       "error",
-      { "functions": false, "variables": false }
+      "always"
     ],
-    "@typescript-eslint/typedef": "error",
-    "@typescript-eslint/type-annotation-spacing": "error",
-    "@typescript-eslint/semi": "error",
-    "@typescript-eslint/no-var-requires": "off",
-    "no-useless-escape": "off",
-    "spaced-comment": "error",
-    "curly": ["error", "multi-line"],
-    "eol-last": ["error", "always"],
+    "linebreak-style": "off",
```

Comment:
Yes! I think I wasn't getting the syntax for this right so I turned it off 😅 fixed.

Reply:
> Since we're forcing windows-style line endings in `.gitattributes`, should we enable that here?

Why is windows-style forced? JSS is supposed to be a cross-platform solution